mppm/about.md

```diff
@@ -6,6 +6,9 @@ description: Overview of Multiplayer Play Mode
 
 Use Multiplayer Play Mode to test multiplayer functionality within the Unity Editor. You can simulate up to four players (the Main Editor and three Virtual Players) simultaneously on the same development device while using the same source assets on disk. Multiplayer Play Mode can help you create multiplayer development workflows that reduce project build times, run it locally, and test the server-client relationship.
 
+## Compatibility
+Multiplayer Play Mode version 1.1.0 is compatible with Unity Editor versions 6000.0.3f1 or later.
```

Comment:
1.1.0 is very limited; can we say "1.1 and later"?

Reply:
Let's check cieht @kpurcellunity